Fix the Apple clang version cutoff point for a working __has_attribute.
https://savannah.gnu.org/bugs/?63866
--- gettext-runtime/config.h.in	2023-11-19 14:22:34.000000000 -0600
+++ gettext-runtime/config.h.in	2024-01-29 02:29:09.000000000 -0600
@@ -1421,7 +1421,7 @@
 #if (defined __has_attribute \
      && (!defined __clang_minor__ \
          || (defined __apple_build_version__ \
-             ? 6000000 <= __apple_build_version__ \
+             ? 7000000 <= __apple_build_version__ \
              : 5 <= __clang_major__)))
 # define _GL_HAS_ATTRIBUTE(attr) __has_attribute (__##attr##__)
 #else
--- gettext-runtime/gnulib-lib/cdefs.h	2023-09-18 15:35:08.000000000 -0500
+++ gettext-runtime/gnulib-lib/cdefs.h	2024-01-29 02:29:09.000000000 -0600
@@ -42,7 +42,7 @@
 #if (defined __has_attribute \
      && (!defined __clang_minor__ \
          || (defined __apple_build_version__ \
-             ? 6000000 <= __apple_build_version__ \
+             ? 7000000 <= __apple_build_version__ \
              : 3 < __clang_major__ + (5 <= __clang_minor__))))
 # define __glibc_has_attribute(attr) __has_attribute (attr)
 #else
--- gettext-runtime/intl/config.h.in	2023-11-19 14:22:28.000000000 -0600
+++ gettext-runtime/intl/config.h.in	2024-01-29 02:29:09.000000000 -0600
@@ -1151,7 +1151,7 @@
 #if (defined __has_attribute \
      && (!defined __clang_minor__ \
          || (defined __apple_build_version__ \
-             ? 6000000 <= __apple_build_version__ \
+             ? 7000000 <= __apple_build_version__ \
              : 5 <= __clang_major__)))
 # define _GL_HAS_ATTRIBUTE(attr) __has_attribute (__##attr##__)
 #else
--- gettext-runtime/libasprintf/config.h.in	2023-11-19 14:22:30.000000000 -0600
+++ gettext-runtime/libasprintf/config.h.in	2024-01-29 02:29:09.000000000 -0600
@@ -691,7 +691,7 @@
 #if (defined __has_attribute \
      && (!defined __clang_minor__ \
          || (defined __apple_build_version__ \
-             ? 6000000 <= __apple_build_version__ \
+             ? 7000000 <= __apple_build_version__ \
              : 5 <= __clang_major__)))
 # define _GL_HAS_ATTRIBUTE(attr) __has_attribute (__##attr##__)
 #else
--- gettext-tools/config.h.in	2023-11-19 14:23:03.000000000 -0600
+++ gettext-tools/config.h.in	2024-01-29 02:29:09.000000000 -0600
@@ -2986,7 +2986,7 @@
 #if (defined __has_attribute \
      && (!defined __clang_minor__ \
          || (defined __apple_build_version__ \
-             ? 6000000 <= __apple_build_version__ \
+             ? 7000000 <= __apple_build_version__ \
              : 5 <= __clang_major__)))
 # define _GL_HAS_ATTRIBUTE(attr) __has_attribute (__##attr##__)
 #else
--- gettext-tools/gnulib-lib/cdefs.h	2023-09-18 15:35:52.000000000 -0500
+++ gettext-tools/gnulib-lib/cdefs.h	2024-01-29 02:29:09.000000000 -0600
@@ -42,7 +42,7 @@
 #if (defined __has_attribute \
      && (!defined __clang_minor__ \
          || (defined __apple_build_version__ \
-             ? 6000000 <= __apple_build_version__ \
+             ? 7000000 <= __apple_build_version__ \
              : 3 < __clang_major__ + (5 <= __clang_minor__))))
 # define __glibc_has_attribute(attr) __has_attribute (attr)
 #else
--- gettext-tools/libgrep/cdefs.h	2023-09-18 15:36:21.000000000 -0500
+++ gettext-tools/libgrep/cdefs.h	2024-01-29 02:29:09.000000000 -0600
@@ -42,7 +42,7 @@
 #if (defined __has_attribute \
      && (!defined __clang_minor__ \
          || (defined __apple_build_version__ \
-             ? 6000000 <= __apple_build_version__ \
+             ? 7000000 <= __apple_build_version__ \
              : 3 < __clang_major__ + (5 <= __clang_minor__))))
 # define __glibc_has_attribute(attr) __has_attribute (attr)
 #else
--- libtextstyle/config.h.in	2023-11-19 14:22:53.000000000 -0600
+++ libtextstyle/config.h.in	2024-01-29 02:29:09.000000000 -0600
@@ -1392,7 +1392,7 @@
 #if (defined __has_attribute \
      && (!defined __clang_minor__ \
          || (defined __apple_build_version__ \
-             ? 6000000 <= __apple_build_version__ \
+             ? 7000000 <= __apple_build_version__ \
              : 5 <= __clang_major__)))
 # define _GL_HAS_ATTRIBUTE(attr) __has_attribute (__##attr##__)
 #else
